### PR TITLE
PR Cl-159 create getsystemwallet controller

### PR DESCRIPTION
**Pull request by kovalenko-Max**

### CODE REVIEW

**Review by Nhevgevgev — APPROVED** [review]

**Review by dneprpi — APPROVED** [review]

File: CRMLite.TransactionStoreCore/Interfaces/IServices/IWalletService.cs [hunk 1]
```diff
@@ -11,5 +11,6 @@ public interface IWalletService
         Task<IEnumerable<Wallet>> GetAllWalletsByLeadIDAsync(Guid leadID);
         Task<Wallet> GetWalletByIDAsync(Guid id);
         Task<Wallet> GetUSDWalletByLeadIDAsync(Guid leadID);
+        Task<Wallet> GetSystemUSDmWalletAsync();
```
Comment (MykhailoKnysh) [inline CRMLite.TransactionStoreCore/Interfaces/IServices/IWalletService.cs]: USD__m__

File: CRMLite.TransactionStoreDatabase/CRMLite/StoredProcedures/Wallet/GetPayPalSystemWallet.sql [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+	C.ID,
+	C.Title,
+	C.Code
 	from [CRMLite].[Wallets] W
-	join [CRMLite].[Balance] B on W.ID = B.WalletID
-
-	where B.LeadID = '471A2146-82BC-4C18-9425-2D8CC89F8540'
\ No newline at end of file
+	join [CRMLite].[Currencies] C on C.ID = W.CurrencyID
+	where W.ID = 'E04B8052-BF0C-41A2-A0FE-5CC331E95E17'
```
Comment (MykhailoKnysh) [inline CRMLite.TransactionStoreDatabase/CRMLite/StoredProcedures/Wallet/GetPayPalSystemWallet.sql:12]: ID as parameter?

File: CRMLite.TransactionStoreDatabase/CRMLite/StoredProcedures/Wallet/GetSystemUSDWallet.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	SELECT 
+	W.ID,
+	W.CurrencyID,
+	W.Amount,
+	C.ID,
+	C.Title,
+	C.Code
+	from [CRMLite].[Wallets] W
+	join [Currencies] C on C.ID = W.CurrencyID
+	where W.ID = '8488F45B-DD2F-41C2-87D4-D9161EA26CDB'
```
Comment (MykhailoKnysh) [inline CRMLite.TransactionStoreDatabase/CRMLite/StoredProcedures/Wallet/GetSystemUSDWallet.sql:12]: ID as parameter?

File: CRMLite.TransactionStoreCore/Interfaces/IServices/IWalletService.cs (hunk 1 above)
Comment (kovalenko-Max) [reply CRMLite.TransactionStoreCore/Interfaces/IServices/IWalletService.cs]: Done

**Review by MykhailoKnysh — APPROVED** [review]

**Pull request merged**